x-pack/solutions/security/plugins/security_solution/server/endpoint/services/workflow_insights/index.ts

```diff
@@ -251,13 +232,65 @@ class SecurityWorkflowInsightsService {
     return this._endpointContext;
   }
 
+  /**
+   * Plugin callbacks called from elastic_assistant plugin
+   */
+
   private registerDefendInsightsCallbacks(
     registerCallback: (callbackId: CallbackIds, callback: Function) => void
   ) {
     registerCallback(
       CallbackIds.DefendInsightsPostCreate,
       this.createFromDefendInsights.bind(this)
     );
+    registerCallback(CallbackIds.DefendInsightsPreCreate, this.onBeforeCreate.bind(this));
+    registerCallback(CallbackIds.DefendInsightsPostFetch, this.onAfterFetch.bind(this));
+  }
+
+  public async onAfterFetch(
```

Comment:
Can you please add response Types to all methods/functions so they are properly typed

Reply:
Added in 96b26b398167f4b2114bdc1e143b5f55a61a52e9